mlir/lib/Dialect/Rock/IR/MfmaInsnGroup.cpp

```diff
@@ -218,6 +218,8 @@ auto getMfmaInsnGroupAttrMap = []() {
                     {ROCDL::mfma_f32_32x32x4bf16::getOperationName()}},
                    {{MfmaTypeId::Bf16TyId, 16, 16},
                     {ROCDL::mfma_f32_16x16x8bf16::getOperationName()}},
+                   {{MfmaTypeId::I8TyId, 64, 64},
+                    {ROCDL::mfma_i32_32x32x8i8::getOperationName()}},
```

Comment:
I'm surprised I have missed this xdlops in the past. Just double check: This is available in all gfx908+ architectures, right?

Reply:
Yes, this was already present in our database. What was missing was getting tiles bigger then `32x32` so to let the warps iterate over more tiles (i.e., repeats > 1)